kratos/tests/cpp_tests/utilities/test_variable_utils.cpp

```diff
@@ -59,12 +59,13 @@ KRATOS_TEST_CASE_IN_SUITE(VariableUtilsSetHistoricalVariablesToZero, KratosCoreF
 
     // Values are properly allocated
     const double tolerance = 1.0e-12;
+    const auto ZeroArray = array_1d<double, 3>(3, 0.0);
     for (const auto& r_node : r_test_model_part.Nodes()) {
-        KRATOS_CHECK_NEAR(r_node.FastGetSolutionStepValue(PRESSURE), 0.0, tolerance);
-        KRATOS_CHECK_NEAR(r_node.FastGetSolutionStepValue(TEMPERATURE), 0.0, tolerance);
-        KRATOS_CHECK_VECTOR_NEAR(r_node.FastGetSolutionStepValue(VELOCITY), ZeroVector(3), tolerance);
-        KRATOS_CHECK_VECTOR_NEAR(r_node.FastGetSolutionStepValue(DISPLACEMENT), ZeroVector(3), tolerance);
-        KRATOS_CHECK_MATRIX_NEAR(r_node.FastGetSolutionStepValue(DEFORMATION_GRADIENT), ZeroMatrix(0,0), tolerance);
+        KRATOS_EXPECT_NEAR(r_node.FastGetSolutionStepValue(PRESSURE), 0.0, tolerance);
```

Comment:
It was really necessary to change the checks?, it would be smarter to keep them and calling the new ones, and remove bit a bit, otherwise this comprieses many changes

Reply:
Yep, this check specifically fails if trying to compare boost vectors (see https://github.com/KratosMultiphysics/Kratos/pull/11506/commits/8c34e8893df63d25c803ecbb542b75758b4a317b)

Reply:
Okay

Reply:
Can we keep a tranition interface, think thatthis will break many branches and independent projects using Kratos

Reply:
I don't see how, I've tried. As I commented in the @KratosMultiphysics/technical-committee there is currently missuse of the `KRATOS_CHECK_XXX` macros, as they are used both as a pseudo-assertions in the tests, and as checks in the rest of the code. 

Since gtest uses full fledged assertions for the test checks, I cannot use the same macro for both, so either we change the KRATOS_CHECK in the code to something else (either be another macro or a proper check) or we make another interface for the assertions based on the existing one (what I did)

If you see any other solution tell me and I will implement it.

Reply:
In that case we can change the macro from CHECK to ASSERT in a independent PR, so this PR is not 250 changes and that way other projects can be adapted

Reply:
The problem is that it will still be a 250+ files changed PR, but as you like.

Reply:
Yes, but it will be a simple change-> Just renaming macro. Here we have renaming + gtest + runkratos + ...